### PR TITLE
[ARO] Add Public, Private to params for help with ingress/apiserver visibility

### PR DESCRIPTION
**Pull request by troy0820**
Add Public, Private to params for help with ingress/apiserver visibility

**Description**  

Add to Public, Private to describe parameters accepted to --apiserver-visibility and ingress-visibility 
`src/azure-cli/azure/cli/command_modules/aro/_params.py`.
Add an example to `src/azure-cli/azure/cli/command_modules/aro/_help.py` to demonstrate how to create a private cluster when executing `az aro create --help`
Resolves https://github.com/Azure/azure-cli/issues/13530

**Testing Guide**  

None

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S170

File: src/azure-cli/azure/cli/command_modules/aro/_params.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ def load_arguments(self, _):
                    validator=validate_worker_count)
 
         c.argument('apiserver_visibility',
-                   help='API server visibility.',
+                   help='API server visibility. Allowed Values: Private, Public',
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/aro/_params.py:73]: Maybe enum type is better. arg_type=get_enum_type(['Private', 'Public'])

**Comment by haroldrandom** [comment]
Do you want this change recorded in HITSOTRY.rst ? If so, please change the title from `{ARO}` to `[ARO]`.

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**ARO**
+
+* Add Public and Private to params for help with ingress/apiserver visibility (#13550)
+
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:9]: You no longer need to modify HISTORY.rst manually. It will be generated from PR title or description.

Comment (troy0820) [reply src/azure-cli/HISTORY.rst:9]: I can revert this commit to allow this to be generated.

**Pull request merged**